### ISSUE TITLE
Composant header avec un seul élément dans la liste quickAccessItems

### ISSUE THREAD

**Issue by humchaop** [codegouvfr/react-dsfr#218]
Suite à un audit d'accessibilité nous avons une non conformité parce qu'un seul élément est présent dans une liste `<ul>` (Voir l'exemple de code fourni). Cela concerne la liste quickAccessItems du composant Header.

![Exemple de Header avec un seul item](https://github.com/codegouvfr/react-dsfr/assets/156206391/1a880c9a-baa6-41c9-a93e-27e1ca5cd89d)

Les listes Html devant comporter plusieurs éléments, il faudrait supprimer cette liste dans le cas où quickAccessItems ne contient qu'un seul élément.

**Comment by enguerranws** [comment]
@humchaop Je suis d'accord (et on a le même souci). On a géré ce cas pour les ButtonsGroup. Je check côté DSFR voir ce qui est proposé. 

@garronej tu as eu un retour de leur part sur ce sujet ou je les contacte ?

**Comment by enguerranws** [comment]
Pour info, de notre côté, on fait tourner [Axe](https://github.com/dequelabs/axe-core), qui ne voit pas de souci sur ce sujet.

**Comment by revolunet** [comment]
C'est un audit RGAA ? je ne vois pas [ce critère pour les listes](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests)

**Comment by humchaop** [comment]
oui c'était bien un audit.
Concernant cette issue il y a eu une mise à jour dans un autre ticket. Je remets l'info ici : 

![image](https://github.com/codegouvfr/react-dsfr/assets/156206391/21e6fe58-4740-482b-9d6d-ae0159966951)

**Comment by enguerranws** [comment]
J'ai posé la question côté DSFR pour savoir ce qu'on fait à ce sujet. Le vrai souci, c'est que de leur côté, ils ne précisent pas de règle pour ce cas (et le DOM associé).
